site/_variables.yml

```
@@ -1,19 +1,44 @@
-# Product names
-vm_platform: "ValidMind Platform UI"
-vm_framework: "ValidMind Developer Framework"
-vm_ui: "platform UI"
-vm_dev: "developer framework"
-vm_risk: "ValidMind AI risk platform"
-
-# Product URLs
+# VARIABLE NAMING CONVENTIONS
+# - Variables for long & short forms should be related (e.g. `developer_framework`, `developer`)
+# - Variable names should be descriptive and consistent (e.g. `user_first_name`, `user_last_name`)
+# - Use prefixes / suffixes for grouping (e.g. `orderDate`, `orderAmount`)
+# - Use snake case (e.g. `developer_framework`)
+# - Avoid abbreviations (e.g. `customer_address` NOT `custAddr`) — `vm` for ValidMind is one exception
+# - Include enough contextual information (e.g. `invoiceTotal` NOT `total`)
+
+
+
+# PRODUCT NAMES
+# Names of ValidMind products or core features
+
+# Long form
+validmind_platform: "ValidMind Platform UI"
+validmind_developer: "ValidMind Developer Framework"
+risk_platform: "ValidMind AI risk platform"
+
+# Short form
+platform: "platform UI"
+developer: "developer framework"
+
+
+# PRODUCT URLS
+# Links to ValidMind hosted web applications
+# This section is always prefaced by `vm_`
+
 vm_url: https://app.prod.validmind.ai
-vm_ui_privatelink: https://private.prod.vm.validmind.ai
-api_host: https://api.dev.vm.validmind.ai/api/v1/tracking
-contact: "[**Contact Us**](https://validmind.com/demo/)"
+vm_privatelink: https://private.prod.vm.validmind.ai
+vm_api: https://api.dev.vm.validmind.ai/api/v1/tracking
+vm_contact: "[**Contact Us**](https://validmind.com/contact/)"
+
+
+# HYPERLINKS
+# Constructed hyperlinks to ValidMind hosted web applications or other common links
+
+platform_login: "[ValidMind Platform UI](https://app.prod.validmind.ai)"
+
 
-# Instructions
-vm_login: "[ValidMind Platform UI](https://app.prod.validmind.ai)"
+# ENVIRONMENT 
+# Environment versions
 
-# Environment 
 python-version: "≧3.8 and <3.11"
-version: v2.0.7
+valimind_version: v2.0.7
```

```suggestion
validmind_version: v2.0.7
```

(I believe "ValiMind" is our discount product. 😛 )